applications/mne_scan/plugins/lsladapter/FormFiles/lsladaptersetup.h

```diff
@@ -0,0 +1,153 @@
+* PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+* HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+* NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+* POSSIBILITY OF SUCH DAMAGE.
+*
+*
+* @brief    Contains the declaration of the LSLAdapterSetup class.
+*
+*/
+
+#ifndef LSLADAPTERSETUP_H
+#define LSLADAPTERSETUP_H
+
+
+//*************************************************************************************************************
+//=============================================================================================================
+// INCLUDES
+//=============================================================================================================
+
+#include "../ui_lsladaptersetup.h"
+
+#include <lsl_cpp.h>
+
+
+//*************************************************************************************************************
+//=============================================================================================================
+// QT INCLUDES
+//=============================================================================================================
+
+#include <QtWidgets>
+
+
+//*************************************************************************************************************
+//=============================================================================================================
+// DEFINE NAMESPACE LSLADAPTERPLUGIN
+//=============================================================================================================
+
+namespace LSLADAPTERPLUGIN
+{
+
+
+//*************************************************************************************************************
+//=============================================================================================================
+// FORWARD DECLARATIONS
+//=============================================================================================================
+
+
+//=============================================================================================================
+/**
+* DECLARE CLASS LSLAdapterSetup
+*
+* @brief The LSLAdapterSetup class provides the LSLAdapter configuration window.
+*/
+class LSLAdapterSetup : public QWidget
+{
+    Q_OBJECT
+
+public:
+
+    //=========================================================================================================
+    /**
+    * Constructs a LSLAdapterSetup which is a child of parent.
+    *
+    * @param [in] parent pointer to potential parent widget
+    */
+    LSLAdapterSetup(int initialBlockSize, QWidget *parent = Q_NULLPTR);
+
+    //=========================================================================================================
+    /**
+    * Destructor of LSLAdapterSetup: default
+    */
+    ~LSLAdapterSetup() = default;
+
+signals:
```

Comment:
Signals go after public, protected and private. Also please remove the slot syntax.